src/mixins/itext_behavior.mixin.js

```diff
@@ -696,12 +685,9 @@
         options.dropTarget = this;
         //  finalize
         this.insertChars(insert, styles, insertAt);
-        this.selectionStart = Math.min(insertAt + selectionStartOffset, this._text.length);
```

Comment:
I disagree with this deletion

Reply:
At least set the selection value to the drop cursor value
Though i still think i should remain as is and the dev csn override it. It's the expected behavior IMO

Reply:
this is outdated, not sure why github doesn't get it